src/components/institution/SelectInstitutionForm.tsx

```diff
@@ -10,19 +10,21 @@ import {
   Radio,
   RadioGroup,
   TextField,
+  Typography,
 } from '@mui/material';
 import { Field, FieldProps, Form, Formik, FormikProps } from 'formik';
 import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { CristinApiPath } from '../../api/apiPaths';
+import i18n from '../../translations/i18n';
```

Comment:
I stedet for å importere en statisk `i18n` her, kan du bruke den som ligger i hooken, sånn at teksten endrer seg med en gang bruker bytter språk: `const { t, i18n } = useTranslation()`

Reply:
Kanskje er det feks en generell `<OrganizationRenderOption />` (elns) vi ønsker å bruke som renderOption alle plasser